server/src/test/java/io/crate/expression/scalar/ParseURIFunctionTest.java

```diff
@@ -0,0 +1,97 @@
+/*
+ * Licensed to Crate.io GmbH ("Crate") under one or more contributor
+ * license agreements.  See the NOTICE file distributed with this work for
+ * additional information regarding copyright ownership.  Crate licenses
+ * this file to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.  You may
+ * obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ *
+ * However, if you have executed another commercial license agreement
+ * with Crate these terms will supersede the license and you may use the
+ * software solely pursuant to the terms of the relevant commercial agreement.
+ */
+
+package io.crate.expression.scalar;
+
+import org.junit.Test;
+
+import io.crate.common.collections.MapBuilder;
+
+import java.util.Map;
+
+public class ParseURIFunctionTest extends ScalarTestCase {
+
+    @Test
+    public void test_null_input() {
+        assertEvaluateNull("parse_uri(null)");
+    }
+
+    @Test
+    public void test_parse_uri() {
+        String uri = "https://crate.io/index.html";
+        Map<String, Object> value = MapBuilder.<String, Object>newMapBuilder()
+            .put("scheme", "https")
+            .put("userinfo", null)
+            .put("hostname", "crate.io")
+            .put("port", null)
+            .put("path", "/index.html")
+            .put("query", null)
+            .put("fragment", null)
+            .map();
+        assertEvaluate(String.format("parse_uri('%s')", uri), value);
```

Comment:
let's use `Locale.ENGLISH` for formatting to be consistent with the codebase

Reply:
done